biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1947,7 +1967,7 @@ end subroutine mortality_litter_fluxes
 
   ! ============================================================================
 
-  subroutine create_patch(currentSite, new_patch, age, areap, label)
+  subroutine create_patch(currentSite, new_patch, age, areap, label,nocomp_pft)
```

Comment:
I noticed that none of the calls to create_patch() initialize the nocomp_pft with anything other than unset_int.  Is it needed as an argument here then?

Reply:
nevermind, forgot to expand the github diffs for the files with lots of changes!